tools/topology/topology2/platform/intel/i2s-dumb-amp-echo-ref.conf

```diff
@@ -0,0 +1,73 @@
+#
+# Pcm, Dai instance & route index all start from 200 to avoid collisions
```

Comment:
I have a PR submitted to alsautils https://github.com/alsa-project/alsa-utils/pull/188 to avoid such collisions in the alsatplg compiler instead of having to do this manually.